harness/determined/cli/task.py

```diff
@@ -217,6 +301,105 @@ def logs(args: Namespace) -> None:
                     *common_log_options,
                 ],
             ),
+            Cmd(
+                "create",
+                create,
+                "create task (EXPERIMENTAL: This command should only be used in dev environments)",
+                [
+                    Arg("config_file", type=FileType("r"), help="task config file (.yaml)"),
+                    Arg(
+                        "--context",
+                        "-c",
+                        type=Path,
+                        help=ntsc.CONTEXT_DESC,
+                    ),
+                    Arg(
+                        "-i",
+                        "--include",
+                        action="append",
+                        default=[],
+                        type=Path,
+                        help=ntsc.INCLUDE_DESC,
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
+                    Arg("--config", action="append", default=[], help=ntsc.CONFIG_DESC),
+                    Arg(
+                        "-f",
+                        "--follow",
+                        action="store_true",
+                        help="follow the logs of the task that is created",
+                    ),
+                    Arg("--fork", type=str, help="id of parent task to fork from"),
+                    Arg(
+                        "-p",
+                        "--parent",
+                        type=str,
+                        help="task id of parent task",
+                    ),
+                    Arg(
+                        "--inherit_context",
+                        action="store_true",
+                        help="inherit the context directory of the parent task (--parent flag required)",
+                    ),
+                ],
+            ),
+            Cmd(
+                "config",
+                config,
+                "get config for given task",
+                [
+                    Arg("task_id", type=str, help="ID of task to pull config from"),
+                    Arg(
+                        "--json",
+                        action="store_true",
+                        help="return config in JSON format",
+                    ),
+                ],
+            ),
+            Cmd(
+                "fork",
+                fork,
+                "fork task",
+                [
+                    Arg("parent_task_id", type=str, help="Id of parent task to fork from"),
+                    Arg(
+                        "-f",
+                        "--follow",
+                        action="store_true",
+                        help="follow the logs of the task that is created",
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
```

Comment:
General feature question:  Are forked tasks allowed to be in different projects than their parents?

Reply:
we want entire task trees to be in one workspace/project (parent & children tasks with respect to supervision hierarchy relationship)
however a forked task can be created in another workspace/project, as it starts a new tree (we used to call this relationship "parent" for experiments, but changed to "forked from" here).